js/viz/funnel.d.ts

```diff
@@ -57,13 +58,14 @@ export {
     Palette,
     PaletteExtensionMode,
     TextOverflow,
+    ShiftLabelOverlap,
     WordWrap,
 };
 
 /** @public */
 export type FunnelAlgorithm = 'dynamicHeight' | 'dynamicSlope';
 /** @public */
```

Comment:
```suggestion
/**
 * @deprecated Use ShiftLabelOverlap from 'devextreme/common/charts' instead 
 */
```

We should deprecate old types `FunnelLabelOverlap`, `PieChartLabelOverlap` to notify users about change and recommend using of `ShiftLabelOverlap`

Do it for PieChart also please.

Let's also add a card for deprecation remove in 2 releases.

Reply:
done